app/models/ValuationMethod.scala

```diff
@@ -50,9 +50,11 @@ object ValuationMethod extends Enumerable.Implicits {
 
   def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
     case (value, index) =>
+      val x = messages(s"valuationMethod.${value.toString}")
+
       RadioItem(
         content = HtmlContent(
-          Html(messages(s"valuationMethod.${value.toString}"))
+          Html(s"<b>${messages(s"valuationMethod.${value.toString}")}</b>")
         ),
```

Comment:
I think you can remove the `x` line here

Reply:
Done